yt/yt/server/all/main.cpp

```diff
@@ -49,6 +61,15 @@ void TryProgram(int argc, const char** argv, const TString& nameSuffix)
 
 int main(int argc, const char** argv)
 {
+    // Shift arguments to handle "program-name" in first argument.
```

Comment:
I'd suggest to add `Allows to provide program name as an argument, for example, ./ytserver-all ytserver-clock`.